smokerd.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=GoodData Smoke testing daemon
+
+[Service]
+Type=simple
+
+Environment=CONFIG=/etc/smokerd/smokerd.yaml
+Environment='OPTIONS=-v --foreground --no-console'
```

Comment:
spawning child processes by smoker is harmless as far as they keep link to parent process (i.e. if they won't daemonize).